openshift/clowder/clowd-app.yaml

```diff
@@ -28,6 +28,12 @@ objects:
             value: 'localhost'
           - name: WITH_MIGRATIONS
             value: '1'
+          - livenessProbe:
+              httpGet:
+                path: /api/automation-hub/_healthz
+                port: 5001
```

Comment:
running at a different port?
Clowder expect port to be existing `webService` port

Reply:
Updated